res/skins/Tango (64 Samplers)/skin.xml

```diff
@@ -38,61 +38,66 @@
     <attributes>
       <!-- Set skin defaults. -->
       <attribute config_key="[Master],num_decks">4</attribute>
-      <attribute config_key="[Master],num_samplers">64</attribute>
-      <attribute persist="true" config_key="[Master],show_mixer">1</attribute>
-      <attribute persist="true" config_key="[Master],show_eqs">1</attribute>
-      <attribute persist="true" config_key="[Master],show_4decks">0</attribute>
-      <attribute persist="true" config_key="[Master],show_4effectunits">0</attribute>
-      <attribute config_key="[Library],show_library">1</attribute>
```

Comment:
`[Skin],show_library`

Reply:
does omitting this CO here affect anything if it isn't used in the here (in this skin)?

Reply:
I don't think so. But it does need to be added back to Deere.

Reply:
added back